lib/discount.js

```diff
@@ -0,0 +1,59 @@
+class Discount {
+	constructor(){
+	}
+	eval(dfr){
+		return undefined;
+	}//dfr = t, where d_t is the discount given to the action t time steps ahead
+}
+
+class GeometricDiscount {//extends Discount{ //Includes no discount with gamma = 1
+	constructor(gamma){
+		this.gamma = gamma
+	}
+	eval(dfr){
```

Comment:
can we use a method name other than 'eval'? eval is the name of a reserved (and very bad) keyword in JS. although it is technically OK, but the connotations trigger me :(